src/components/TaskList/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useSelector } from "react-redux";
+import TaskCard from "/src/components/TaskCard";
+import PropTypes from "prop-types";
+import "./index.scss";
+
+const TaskList = () => {
+  const tasks = useSelector(state => state.tasks);
+
+  return (
+    tasks.map((task) => (
+      <TaskCard key={task.id} task={task} />
+    ))
+  );
+};
+
+TaskList.propTypes = {
+  tasks: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      title: PropTypes.string.isRequired,
```

Comment:
Here we have defined `tasks` shape with two property but inside `TaskCard` component we have validated three property. May be we have missed `created_at` here